docs/docs/running-hello-world.md

```diff
@@ -73,216 +75,235 @@ The web server loads the enclave and waits for requests from the client. Amongst
+   This will give you a bash shell in the container that simulates a native Linux machine. You can tweak this command
+   according to the specific needs of your project. Please take a look at the explanation of the options used in this
+   command [at the end of this tutorial](#appendix-summary-of-docker-command-options). Check the
+   [Docker reference manual](https://docs.docker.com/engine/reference/commandline/run/) or
+   run `docker <command> --help` on the Docker command line interface for more information.
 
-    * You can now run the host and client as we did for mock mode. The only differences are that the host must be run from
-     inside the container, and the host JAR will be named `host-simulation.jar` rather than `host-mock.jar`.
-    * Press `CTRL+D` to exit the container when finished.
+3. You can now run the host and client as we did for mock mode. The only differences are that the host must be run from
+   inside the container, and the host JAR will be named `host-simulation.jar` rather than `host-mock.jar`.
+4. Press `CTRL+D` to exit the container when finished.
 
 === "Linux"
-    If the system requirements are satisfied for your desired mode,
-    you can run your app in the same way on a Linux system as you would in mock mode.
 
-!!! note
-    If your platform or container doesn't support SGX enclaves, you should see the following message when you run the host:
+On a Linux machine that meets the [system requirements](enclave-modes.md#system-requirements), you can run your app in
+all the modes the same way as in mock mode.
+
+!!!Note
+
+    If your platform or container doesn't support SGX enclaves, you might see the following message when you run the
+    host:
     ```text
-    This platform does not support hardware enclaves: SGX_DISABLED_UNSUPPORTED_CPU: SGX is not supported by the CPU in this system
+    This platform does not support hardware enclaves: SGX_DISABLED_UNSUPPORTED_CPU: SGX is not supported by the 
+    CPU in this system
     ```
     Don't worry, you will still be able to use simulation mode even if you see this message.
 
 ### Run the client in other modes
 The client build is independent of the mode. The only difference is that the enclave will have a different signing key
-when you are not using mock mode, which will be reflected in the Enclave Constraint provided to the client.
+when you are not using mock mode, which will be reflected in the enclave constraint provided to the client.
 
-When you run the host, you should see the new signing key in the output: 
+1. When you run the host, you should see the new signing key in the output:
 ```bash hl_lines="3"
-[main] INFO com.r3.conclave.host.web.EnclaveWebController - Remote attestation for enclave A92F481B7EEAE42D3EBB162BF77613605AF214D77D2E63D75A610FD485CFD7D6:
+[main] INFO com.r3.conclave.host.web.EnclaveWebController -
+Remote attestation for enclave A92F481B7EEAE42D3EBB162BF77613605AF214D77D2E63D75A610FD485CFD7D6:
   - Mode: MOCK
```

Comment:
We are saying above that we are running in different modes than MOCK and here we mention MOCK which is not correct. Also as code signer is different, this shouldn't be mock mode, but simulation or debug

Reply:
Changed to SIMULATION.